<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfirmForm</class>
 <widget class="QWidget" name="ConfirmForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>941</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>確認画面</string>
  </property>
  <widget class="QTableWidget" name="tableWidgetImageBank">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>241</width>
     <height>811</height>
    </rect>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
  </widget>
  <widget class="QToolButton" name="toolButtonImageBankOnlyNG">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>NGのみ収集</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frameImagePanel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>671</width>
     <height>901</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidgetProperty">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>300</y>
     <width>491</width>
     <height>591</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="AlignmentLarge"/>
   <widget class="QWidget" name="DotColorMatching"/>
   <widget class="QWidget" name="MeasureLineMove"/>
   <widget class="QWidget" name="ColorDifference"/>
   <widget class="QWidget" name="BCRInspection"/>
  </widget>
  <widget class="QPushButton" name="pushButtonWhole">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>91</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>全体表示</string>
   </property>
  </widget>
  <widget class="QFrame" name="frameAlgo">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>0</y>
     <width>451</width>
     <height>34</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMasterImage">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>0</y>
     <width>91</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>マスター画像</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frameControl">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>890</y>
     <width>491</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButtonSetting">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>121</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>設定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClose">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>121</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>閉じる</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frameImageBankControl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>840</y>
     <width>241</width>
     <height>101</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QToolButton" name="toolButtonImageBankUpdate">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>更新</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonImageBankRetryAll">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>全て再検査</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frameRightTop">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>0</y>
     <width>491</width>
     <height>301</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTableWidget" name="tableWidgetResult">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>391</width>
      <height>301</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>Phase</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>場所</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>判定ロジック</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>NG結果</string>
     </property>
    </column>
   </widget>
   <widget class="QListWidget" name="listWidgetPhase">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>101</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
